docs/cave/d2d-perspective.md

```diff
@@ -6,7 +6,7 @@ D2D (Display 2-Dimensions) is the default AWIPS CAVE perspective, designed to mi
 
 System menus include **CAVE**, **File**, **View**, **Options**, and **Tools**.
 
-Data menus include **[Models](d2d-grids)**, **Surface**, **[NCEP/Hydro](d2d-hydro)**, **[Upper Air](d2d-uair)**, **[Satellite](d2d-satellite)**, **Local Radar Stations**,  **[Radar](d2d-radar)**, **MRMS**, and **Maps**.
```

Comment:
Technically we have a Satellite page, but it is a different link (for GOES 16/17), but we could probably leave the link off...http://unidata.github.io/awips2/cave/goes-16-satellite/

Reply:
I think we're okay leaving the Satellite link off for now.  Maybe after we put out this release if we want to take a stab at developing a page for each of the data menus then we can, and it will be consistent for all of them?

Reply:
Sounds good